arm-sql/2015-05-01-preview/examples/ServerSecurityAlertCreateMax.json

```diff
@@ -0,0 +1,41 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "securityalert-4799",
+    "serverName": "securityalert-6440",
+    "securityAlertPolicyName": "Default",
+    "api-version": "2015-05-01-preview",
+    "serverSecurityAlertPolicy": {
+        "properties": {
+            "state": "Enabled",
+            "emailAccountAdmins": "Enabled",
+            "emailAddresses": "testSecurityAlert@microsoft.com",
```

Comment:
The property name is in plural form "emailAddress **es**" and it only accepts one email address as a string. An array would have been better.